### ISSUE TITLE
quodlibet fails to start with python 3.10

### ISSUE THREAD

**Issue by hannes101** [quodlibet/quodlibet#3818]
Steps to reproduce
------------------
Current fedora 35, which is not yet stable. Please see downstream bug report. I have not yet upgraded to it, so for me it's hard to debug.
https://bugzilla.redhat.com/show_bug.cgi?id=2008422

Expected Output
---------------
quodlibet starting without an issue.

Actual Output
-------------
quodlibet fails to start

Test System
-----------

### Which version of Quod Libet?
4.4.0

### Which operating system 
fedora 35

**Comment by declension** [comment]
I think this should be fixed by #3800 so should be included in next release.

Please re-open if this doesn't fix it though

**Comment by hannes101** [comment]
Thanks a lot, then I am all for a new release, but can also include the commit as a patch 👍

**Comment by elcste** [comment]
In Jammy Jelly, the Ubuntu 22.04 LTS release coming out next month, the defaults is python 3.10, so Quod Libet installed from the Ubuntu package repo doesn't start up 😱 I just reported it to Ubuntu (https://bugs.launchpad.net/ubuntu/+source/quodlibet/+bug/1964791), although I suspect it should go through Debian, but I don't have a Debian install to confirm the issue there.

**Comment by linux4me** [comment]
Ubuntu 22.04 LTS (Jammy) is out, and Quod LIbet is still broken:
```
E: 0.164: errorreport.main.errorhook: collection.py:27:<module>: ImportError: cannot import name 'Iterable' from 'collections' (/usr/lib/python3.10/collections/__init__.py)
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/quodlibet/errorreport/main.py", line 175, in errorhook
    sentry_error = sentry.capture(exc_info, fingerprint=fingerprint)
  File "/usr/lib/python3/dist-packages/quodlibet/errorreport/sentrywrapper.py", line 247, in capture
    client = Client(
  File "/usr/lib/python3/dist-packages/quodlibet/packages/raven/base.py", line 224, in __init__
    from raven.context import Context
  File "/usr/lib/python3/dist-packages/quodlibet/_import.py", line 47, in load_module
    mod = importlib.import_module(loadname)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/usr/lib/python3/dist-packages/quodlibet/packages/raven/context.py", line 10, in <module>
    from collections import Mapping, Iterable
ImportError: cannot import name 'Mapping' from 'collections' (/usr/lib/python3.10/collections/__init__.py)

Original exception was:
Traceback (most recent call last):
  File "/usr/bin/quodlibet", line 14, in <module>
    sys.exit(main())
  File "/usr/lib/python3/dist-packages/quodlibet/main.py", line 44, in main
    import quodlibet.library
  File "/usr/lib/python3/dist-packages/quodlibet/library/__init__.py", line 20, in <module>
    from quodlibet.library.libraries import SongFileLibrary, SongLibrary
  File "/usr/lib/python3/dist-packages/quodlibet/library/libraries.py", line 32, in <module>
    from quodlibet.util.collection import Album
  File "/usr/lib/python3/dist-packages/quodlibet/util/collection.py", line 27, in <module>
    from collections import Iterable
ImportError: cannot import name 'Iterable' from 'collections' (/usr/lib/python3.10/collections/__init__.py)
```

**Comment by elcste** [comment]
@linux4me Version 4.5 is in Debian Sid, but it can too late to get auto-synced to Ubuntu 22.04 before the release. There is an [Ubuntu bug](https://bugs.launchpad.net/ubuntu/+source/quodlibet/+bug/1964791) asking to get it updated into 22.04. 

In the mean time, the easiest workaround is to manually download and install the [Debian package](https://packages.debian.org/sid/all/quodlibet/download) (and [Ex Falso](https://packages.debian.org/sid/all/exfalso/download) if you want). If/when Ubuntu gets these packages, they will automatically install with other updates since the Ubuntu version number will be higher.

**Comment by linux4me** [comment]
@elcste Thanks so much for the bug info and the workaround!

**Comment by elcste** [comment]
Update for Ubuntu 22.04: [an updated 4.4.0 package](https://bugs.launchpad.net/ubuntu/+source/quodlibet/+bug/1964791/comments/13) was just released for 22.04 that includes a cherry-picked fix for the Python 3.10 startup issue.

Additionally, Ubuntu Kinetic (development release that will become 22.10) now includes the 4.5 package from Debian.
